README.md

```diff
@@ -40,58 +40,60 @@ This collection provides a list of educational resources for Software Engineers.
 
 ## The Software Development section consists of :
 
-### 1. Web Development - Here are just 5 from our huge range :
+### 1. Web Development - Here are just 6 from our huge range :
```

Comment:
After you revert the w3 resource, please change this back to 5, not 6 @kenhorlador . Thank you.